packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -213,8 +218,41 @@ class SnackBar extends StatelessWidget {
     }
     final CurvedAnimation heightAnimation = new CurvedAnimation(parent: animation, curve: _snackBarHeightCurve);
     final CurvedAnimation fadeAnimation = new CurvedAnimation(parent: animation, curve: _snackBarFadeCurve, reverseCurve: const Threshold(0.0));
+    Widget snackbar = new SafeArea(
+      top: false,
+      child: new Row(
+        children: children,
+        crossAxisAlignment: CrossAxisAlignment.center,
+      ),
+    );
+    snackbar = new Semantics(
+      container: true,
+      liveRegion: true,
+      onDismiss: () {
+        Scaffold.of(context).removeCurrentSnackBar(reason: SnackBarClosedReason.dismiss);
+      },
+      child: new Dismissible(
+        key: const Key('dismissible'),
+        direction: DismissDirection.down,
+        resizeDuration: null,
+        onDismissed: (DismissDirection direction) {
+          Scaffold.of(context).removeCurrentSnackBar(reason: SnackBarClosedReason.swipe);
+        },
+        child: new Material(
+          elevation: 6.0,
+          color: backgroundColor ?? _kSnackBackground,
+          child: new Theme(
+            data: darkTheme,
+            child: mediaQueryData.accessibleNavigation ? snackbar : new FadeTransition(
+              opacity: fadeAnimation,
+              child: snackbar,
+            ),
+          ),
+        ),
+      ),
+    );
     return new ClipRect(
-      child: new AnimatedBuilder(
+      child: mediaQueryData.accessibleNavigation ? snackbar : new AnimatedBuilder(
```

Comment:
Just curious, do we not want the Material in the widget hierarchy in the accessibility case?
e.g when accessibility is on if someone is tapping with 2 fingers do we not want the same ink splashes?

Reply:
Not sure what you mean - the material is still in the tree. Ink splashes will only be disabled if animations are disabled (not done yet).

Reply:
Oops, I misread the assignments above.